extensions/ql-vscode/src/query-history.ts

```diff
@@ -192,7 +192,7 @@ export class QueryHistoryManager {
 
   async handleShowQueryLog(queryHistoryItem: CompletedQuery) {
     if (queryHistoryItem.logFileLocation) {
-      const uri = vscode.Uri.parse(queryHistoryItem.logFileLocation);
+      const uri = vscode.Uri.parse('file:' + queryHistoryItem.logFileLocation);
```

Comment:
Does this work when your log file path contains `#`? (cf the example for the `file` method at https://code.visualstudio.com/api/references/vscode-api#Uri)
If `logFileLocation` is a filesystem path then I think we should be using `vscode.Uri.file`.
If it's a URI then what happens if it already has a scheme?

Reply:
`logFileLocation` is a string. I think the `uri.file` method is the one to use. Thanks for catching that.